assets/components/inlinePaymentLogos/inlinePaymentLogos.scss

```diff
@@ -6,4 +6,94 @@
     margin-left: 4px;
     width: 30px;
   }
-}
\ No newline at end of file
+}
+
+// AB test paymentLogosTest
+.component-inline-payment-logos--one_off-variant {
+  @include mq($from: desktop) {
+    position: relative;
+    top: 222px;
+    left: 36px;
+    float: right;
+    .svg-paypal-logo {
+      position: relative;
+      top: -56px;
+      left: -48px;
+    }
+
+    svg {
+      width: 42px;
+      height: 27px;
+    }
```

Comment:
I think in this case it actually does make sense to style by element because what we have is a container for a load of svgs all of which we want to style identically. The only other ways to do this would be to have duplicated styling for `.svg-visa-logo`, `svg-mastercard-logo` etc. Or to introduce a generic `svg` class on all of the svgs, which is basically a more verbose way of styling by element.